core/src/util/builders.cpp

```diff
@@ -137,6 +137,12 @@ void Builders::buildPolygonExtrusion(const Polygon& _polygon, float _minHeight,
             normalVector = glm::cross(upVector, b - a);
             normalVector = glm::normalize(normalVector);
 
+            if (std::isnan(normalVector.x)
```

Comment:
Should each data sources reflect this? I'm not sure. Testing a NaN number is barely a no-op.